src/build/download_file.mjs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2023 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {createWriteStream} from 'node:fs';
+import {default as redir} from 'follow-redirects';
```

Comment:
In the server we use `node-fetch`, which is nicer because it's the same interface as the browser. I don't know this library you are using, so I need to learn.

It's fine to keep as is, but prefer node-fetch.

Reply:
Good to know that, I can use `node-fetch` instead. `follow-directs` package is just a simple wrapper around Node's `http`/`https`, but with redirection support.

Reply:
Replaced with `node-fetch`.